xcube_sh/store.py

```diff
@@ -149,17 +191,21 @@ def _get_open_data_params_schema(self, dsd: DataDescriptor = None) -> JsonObject
             time_range=JsonArraySchema(items=(JsonStringSchema(format='date-time'),
                                               JsonStringSchema(format='date-time'))),
             # TODO: add pattern
-            time_period=JsonStringSchema(default='1D'),
-            time_tolerance=JsonStringSchema(default=DEFAULT_TIME_TOLERANCE),
+            time_period=JsonStringSchema(default='1D', nullable=True,
+                                         enum=[None,
+                                               *map(lambda n: f'{n}D', range(1, 7)),
+                                               '1W', '2W']),
+            time_tolerance=JsonStringSchema(
+                default=DEFAULT_TIME_TOLERANCE),
```

Comment:
As it is documented that you want a pandas-compatible format, you can leave this as it is, but I'd much appreciate it if you could add an enum (as in the time_period) or a pattern.

Reply:
I'll do!